generic/GenericCplComp.F90

```diff
@@ -1,3 +1,4 @@
+#define DEALOC_(A) if(associated(A))then;if(MAPL_ShmInitialized)then;call MAPL_SyncSharedMemory(rc=STATUS);call MAPL_DeAllocNodeArray(A,rc=STATUS);else;deallocate(A,stat=STATUS);endif;_VERIFY(STATUS);NULLIFY(A);endif
```

Comment:
If we are going to make hideous macros, let's at least make them readable.  You'll need to check that I did not break anything in the edits though.
```suggestion
#define DEALOC_(A) \
    if(associated(A))then \
          if(MAPL_ShmInitialized)then \
              call MAPL_SyncSharedMemory(rc=STATUS) \
              call MAPL_DeAllocNodeArray(A,rc=STATUS) \
           else \
              deallocate(A,stat=STATUS) \
           endif \
       _VERIFY(STATUS) \
       NULLIFY(A) \
    endif
```

Reply:
This is much cleaner! I wish we have done very similar macros the same way.

Reply:
I only recently became aware that this was portable.

Reply:
The intel compiler did not like the new style. Let me try with ; at the end of each line

Reply:
Sorry about that.

Reply:
";" works. I am adopting the new style, and about to run a quick test, and push the changes